src/renderers/shared/utils/Transaction.js

```diff
@@ -229,6 +236,8 @@ var Mixin = {
   },
 };
 
+export type TransactionType = typeof Transaction.Mixin;
```

Comment:
My suggestion is that you change from:

```js
export type TransactionType = typeof Transaction.Mixin;

var Transaction = {
  Mixin: Mixin,
  OBSERVED_ERROR: {},
};

module.exports = Transaction;
```

to

```js
export type Transaction = typeof Mixin;

module.exports = {
  Mixin: Mixin,
  OBSERVED_ERROR: {},
};
```

For now.

That way we can keep calling the type `Transaction`.